Formula/gedit.rb

```diff
@@ -40,11 +40,9 @@ def install
     ENV["DESTDIR"] = "/"
     ENV.append "LDFLAGS", "-Wl,-rpath,#{lib}/gedit" if OS.linux?
 
-    mkdir "build" do
-      system "meson", *std_meson_args, ".."
-      system "ninja", "-v"
-      system "ninja", "install", "-v"
-    end
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build"
```

Comment:
Do we want the same compilation output as `ninja -v` (e.g. `[XXX/XXX] clang -I... `)? Or do we want just the summary steps like `ninja` non-verbose (`[XXX/XXX] Compiling C object ...`)?

Current command is close to running latter, `ninja`

`meson compile -C build -v` should be closer to `ninja -v`

Reply:
Let's stick with the verbose output.